wpmcpp/src/wpmutils.cpp

```diff
@@ -2116,26 +2102,13 @@ QString WPMUtils::moveToRecycleBin(QString dir)
 
 bool WPMUtils::is64BitWindows()
 {
-#ifdef __x86_64__
+#if defined(__x86_64__) || defined(_WIN64)
     return true;
 #else
     // 32-bit programs run on both 32-bit and 64-bit Windows
     // so must sniff
-    BOOL WINAPI (* lpfIsWow64Process_) (HANDLE,PBOOL);

```

Comment:
Because QueryFullProcessImageNameW can be accessed directly if the program links kernel32.lib. If it is not working with mingw, I could change it.